Library/Homebrew/download_strategy.rb

```diff
@@ -352,8 +352,9 @@ def resolve_url_and_basename(url)
       end
     end
 
-    filenames = lines.map { |line| line[/^Content\-Disposition:\s*attachment;\s*filename=(["']?)([^;]+)\1/i, 2] }
-                     .compact
+    filenames =
+      lines.map { |line| line[/^Content\-Disposition:\s*(?:inline|attachment);\s*filename=(["']?)([^;]+)\1/i, 2] }
```

Comment:
Mind using a `do` here and splitting the regex handling onto multiple lines?

Reply:
Multi-line chaining looks ugly with `do … end`.

Reply:
@reitermarkus Cool, fine with me 👍